core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/GraalVM.java

```diff
@@ -186,8 +186,21 @@ public static final class Version implements Comparable<Version> {
         public static final Version VERSION_23_1_0 = new Version("GraalVM 23.1.0", "23.1.0", "21", Distribution.GRAALVM);
         public static final Version VERSION_24_0_0 = new Version("GraalVM 24.0.0", "24.0.0", "22", Distribution.GRAALVM);
 
+        /**
+         * The minimum version of GraalVM supported by Quarkus.
+         * Versions prior to this are expected to cause major issues.
+         */
         public static final Version MINIMUM = VERSION_22_2_0;
```

Comment:
I think this should be `23.0` (JDK 17-based), by now for quarkus main. 22.2.0 is very old. Failing that, use 22.3.0 at least.

Reply:
Why though? By bumping `MINIMUM` we (mandrel team) will no longer be able to do tests with these versions (mostly to see the evolution through time or to see in which version a regression was introduced) even if there is no major "breakage". So far there is no extra effort required on our side to keep this as is. cc @galderz

Reply:
I don't see this being tested anywhere with Quarkus main, so unless you can show this actually works with 22.2, there is no point in keeping it.